src/System.ServiceModel.Security/ref/System.ServiceModel.Security.cs

```diff
@@ -227,4 +234,16 @@ public enum SecurityKeyType
         AsymmetricKey,
         BearerKey
     }
+    public partial class GenericXmlSecurityKeyIdentifierClause : SecurityKeyIdentifierClause
+    {
```

Comment:
You need to declare that the Matches method has been overridden otherwise someone deriving from this class might get wrong behavior if calling base.Matches(). You should probably add the ReferenceXml property too otherwise you can't get the XmlElement back out if you want to.

Reply:
updated